application/web/index.html

```diff
@@ -96,9 +97,21 @@ <h2 id="journal-title"></h2>
                             <button id="save-entry" class="primary">
                                 <span>Save Entry</span>
                             </button>
-                            <button class="three-dots-entry">
-                                <img src="./assets/vdots-entry-black.png" width="24" height="24">
+                            <button id = "three-dots-entry" class="three-dots">
+                                <img src="./assets/vdots-grey.svg" alt="Three Dots" class="dots-icon">
                             </button>
+                            <div id="dropdown-menu-entry" class="dropdown-menu">
+                                <div class="dropdown-item" data-action="markdown">
+                                    <img src="./assets/markdown-icon.svg" alt="Markdown Icon" class="dropdown-icon"> Download as Markdown
+                                </div>
+                                <div class="dropdown-item" id="devModeToggle" data-action="devmode">
+                                    <img src="./assets/devmode-icon.svg" alt="Dev Icon" class="dropdown-icon"> Dev Mode
+                                    <img src="./assets/checkmark.svg" alt="Checkmark Icon" class="checkmark">
+                                </div>
+                                <div class="dropdown-item delete" data-action="delete">
+                                    <img src="./assets/trashcan-red.svg" alt="Delete Icon" class="dropdown-icon"> Delete Entry
+                                </div>
+                            </div>
```

Comment:
Not sure if you want to create a custom element for this and call it <dropdown></dropdown> and put all of those code inside of this element.

If you do that, you will need to adjust your styling, but it's up to you

Reply:
I'm gonna work on this when I do issue #92